packages/components/src/Page/Page.css

```diff
@@ -21,6 +21,10 @@
   margin-bottom: var(--space-base);
 }
 
+.titleBar .subheader {
+  margin-bottom: var(--space-base);
```

Comment:
This is just a visual thing, but I don't think we need the extra spacing here

### With
![image](https://user-images.githubusercontent.com/39704901/105774504-9055c900-5f22-11eb-927a-6a516c774476.png)

### Without
![image](https://user-images.githubusercontent.com/39704901/105774617-b4b1a580-5f22-11eb-92fd-3dcadcb8e5fe.png)

Reply:
I think it feels better with a more even rhythm between

title
subtitle
intro
content

Rather than the larger gap in between them